crowbar_framework/config/experimental.yml

```diff
@@ -0,0 +1,32 @@
+default: &default
+  disallow_restart:
+    enabled: false
```

Comment:
These two lines shouldn't really be there (the feature has not been backported). But I guess it doesn't really hurt either.